modelci/hub/converter/to_trt.py

```diff
@@ -74,23 +74,29 @@ def from_onnx(
 
         # Save TRT engine
         trt_logger = trt.Logger(trt.Logger.WARNING)
-        with trt.Builder(trt_logger) as builder:
-            with builder.create_network(
-                    1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH)) as network:
-                with trt.OnnxParser(network, trt_logger) as parser:
-                    builder.max_workspace_size = GiB(1)  # 1GB
-                    builder.max_batch_size = 1
-                    if int8_calibrator is not None:
-                        builder.int8_mode = True
-                        builder.int8_calibrator = int8_calibrator
-
-                    print('Loading ONNX file from path {}...'.format(onnx_path))
-                    with open(onnx_path, 'rb') as model:
-                        parser.parse(model.read())
-                    engine = builder.build_cuda_engine(network)
-
-                    with open(save_path / 'model.plan', 'wb') as f:
-                        f.write(engine.serialize())
+        explicit_batch = 1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH)
+
+        with trt.Builder(trt_logger) as builder, builder.create_network(explicit_batch) as network, trt.OnnxParser(network, trt_logger) as parser:
+            builder.max_workspace_size = GiB(1)  # 1GB
+            builder.max_batch_size = 1
+            if int8_calibrator is not None:
+                builder.int8_mode = True
+                builder.int8_calibrator = int8_calibrator
+
+            print('Loading ONNX file from path {}...'.format(onnx_path))
```

Comment:
Would it be better to replace `print()` with `logger = Logger('converter', welcome=False)` ?

Reply:
good idea